src/core/windows/SDL_windows.c

```diff
@@ -96,6 +96,9 @@ WIN_CoInitialize(void)
        attribute, which, AFAIK, should initialize COM.
     */
     return S_OK;
+#elif defined(__XBOXONE__) || defined(__XBOXSERIES__)
+    /* On Xbox, there's no need to call CoInitializeEx (and it's not implemented) */
```

Comment:
``CoInitializeEx`` is needed and supported for ``_GAMING_XBOX`` if you use Windows Imaging Component (WIC) or other COM interfaces. DirectX 12 itself on Xbox is *not* COM.